utils/export_supported_formats.py

```diff
@@ -317,32 +276,11 @@ class DataFormatInformationExractor(object):
           'Windows USB storage device data', ''),
       'windows_version': (
           'Windows version information', ''),
-      'winevt': (
-          'Windows Event Log (EVT) file',
-          'libyal:libevt:Windows Event Log (EVT) format'),
-      'winevtx': (
-          'Windows XML Event Log (EVTX) file',
-          'libyal:libevtx:Windows XML Event Log (EVTX)'),
-      'winfirewall': (
-          'Windows Firewall', ''),
-      'winiis': (
-          'Microsoft IIS log file', ''),
-      'winjob': (
-          'Windows Job file (also known as "at jobs"',
-          'dtformats:Job file format'),
       'winlogon': (
           'Windows log-on data', ''),
-      'winrar_mru': (
```

Comment:
I'm expecting a relevant change in winreg_plugins/winrar.py, but I don't see it here

Reply:
Good catch, these changes were made in https://github.com/log2timeline/plaso/commit/a0275e2a748866f2ace3c1f9b71471cf3dee4085 but this one was not removed.